README.md

```diff
@@ -47,46 +29,9 @@ Additional info on Azure deployment models [https://azure.microsoft.com/en-us/do
 
```

Comment:
you're right, I'll leave it as it now as I'm not changing it here, and we can decide on this later. thanks!